apps/src/aichat/views/demoMessages.ts

```diff
@@ -0,0 +1,89 @@
+// Will delete once we've connected UI to backend.
+
+const ASSISTANT = 'assistant' as const;
+const USER = 'user' as const;
+const SYSTEM = 'system' as const;
+const OK = 'ok' as const;
+const PERSONAL = 'personal' as const;
+const INAPPROPRIATE = 'inappropriate' as const;
+
+const demoChatMessages = [
```

Comment:
Hm, I wonder if you declare the type of this array, you may not need to use `as const` on all of those values? Especially if this chat message structure is going to be shared between the front end and the back end, it would probably be valuable to pull it out from the React component and into its own type, stored in something like `src/aichatc/types.ts`. Then you could declare this array as `const demoChatMessages: ChatMessage[] = [ ... ]`.

Reply:
Just added a `ChatMessage` type and no longer need `as const` in demo messages. Thanks!